### ISSUE TITLE
(error) FAILED to execute command: "php" "C:\Program Files\Vemto\resources\static\VMTTL1"

### ISSUE THREAD

**Issue by smp4000** [VemtoOrg/vemto2-issues#116]
**Error Message:**
(error) FAILED to execute command: "php" "C:\Program Files\Vemto\resources\static\VMTTL1"

**Stack:**
```
Error: Command failed: "php" "C:\Program Files\Vemto\resources\static\VMTTL1"

    at ChildProcess.exithandler (node:child_process:407:12)
    at ChildProcess.emit (node:events:527:28)
    at maybeClose (node:internal/child_process:1092:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5)
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.9
win32 10.0.22631
Locale: de
App version: 2.0.9

**Comment by TiagoSilvaPereira** [comment]
Hi @smp4000 are all dependencies green?

![image](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/11933789/bb0e2bf2-d33a-475e-8377-c525462e7b41)

**Comment by TiagoSilvaPereira** [comment]
@smp4000 also please check if Vemto 2.0.10 solves this issue

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.11
